src/index.js

```diff
@@ -71,12 +76,14 @@ const SplunkRum = {
   _deregisterInstrumentations: NOOP,
   DEFAULT_AUTO_INSTRUMENTED_EVENTS,
   init: function (options = {}) {
+    diag.setLogger(new DiagConsoleLogger(), options.debug ? DiagLogLevel.DEBUG : DiagLogLevel.ERROR);
+
     options = Object.assign({}, OPTIONS_DEFAULTS, options, {
       exporter: Object.assign({}, OPTIONS_DEFAULTS.exporter, options.exporter),
     });
 
     if (this.inited) {
-      console.log('SplunkRum already init()ed.');
+      diag.warn('SplunkRum already init()ed.');
```

Comment:
should this be error to match previous behavior of it being output to console regardless of options.debug?

Reply:
Right, that makes sense.